build.sh

```diff
@@ -24,9 +40,11 @@ installer_url () {
 
 build_6x_7x () {
   declare -r version="${1}";
-  declare -r installer_path="racket-minimal-${version}-x86_64-linux-natipkg.sh";
-  declare -r installer=$(installer_url "${version}" "${installer_path}") || exit "${?}";
-  build "racket" "buildpack-deps:stable" "${installer}" "${version}";
+  for variant in "" "-minimal"; do
```

Comment:
Since there's only two cases, could we just skip the loop and write the loop body twice? That way I don't have to check that the `racket${variant}-${version}` part expands correctly, which took me a minute to determine here.

Reply:
Yup. I'll do that.